### PR TITLE
Directly add doctrine/annotations dependency

### PR DESCRIPTION
**Pull request by dbu**
continuation of #36

### CODE REVIEW

**Comment by dbu** [comment]
@Spea now that the lowest version build takes annotations 1, it fails. should we keep registering the annotation loader to keep support version 1? does it hurt in version 2?

**Comment by dbu** [comment]
cs fixer has not been running in the ci before. need to adjust a couple of things to comply with the latest rules.

File: src/Metadata/PropertyAccessor.php [hunk 1]
```diff
@@ -29,7 +23,7 @@ public static function none(): self
 
     public function isDefined(): bool
     {
-        return $this->hasGetterMethod() || $this->hasGetterMethod();
+        return $this->hasGetterMethod() || $this->hasSetterMethod();
```
Comment (dbu) [inline src/Metadata/PropertyAccessor.php:26]: thank you phpstan :+1:

**Comment by dbu** [comment]
> @Spea now that the lowest version build takes annotations 1, it fails. should we keep registering the annotation loader to keep support version 1? does it hurt in version 2?

seems that some other error was causing that, it now is green so we don't need to explicitly load the annotation loader.

**Comment by Spea** [comment]
@dbu Awesome, thank you for your help on this, really appreciated!

**Pull request merged**